Open Judge System/Web/OJS.Web/Areas/Administration/ViewModels/Problem/DeleteProblemViewModel.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace OJS.Web.Areas.Administration.ViewModels.Problem
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.Linq.Expressions;
+    using System.Web.Mvc;
+
+    using OJS.Common.Extensions;
+    using OJS.Data.Models;
+
+    using Resource = Resources.Areas.Administration.Problems.ViewModels.DetailedProblem;
+
+    public class DeleteProblemViewModel
+    {
+        public static Expression<Func<Problem, DeleteProblemViewModel>> FromProblem =>
+            problem => new DeleteProblemViewModel
+            {
+                Id = problem.Id,
+                ContestId = problem.ContestId,
+                Name = problem.Name,
+                MaximumPoints = problem.MaximumPoints,
+                TimeLimit = problem.TimeLimit,
+                MemoryLimit = problem.MemoryLimit,
+                SourceCodeSizeLimit = problem.SourceCodeSizeLimit,
+                Checker = problem.Checker.Name,
+                OrderBy = problem.OrderBy,
+                SolutionSkeletonData = problem.SolutionSkeleton
```

Comment:
Do we need all of this data in the confirm delete window? What purpose does it serve to the user to know what's the checker for the problem he/she wants to delete for example?